### PR TITLE
Several improvements

### PR DESCRIPTION
**Pull request by strengejacke**
- selecte NSE handles group-var from grouped df
- add `append` to more methods (`normalize()`, ...)
- where we have an `append` argument, the `.grouped_df` methods need to set `append = FALSE` when calling regular data frame methods
- removes unnecessary formula-evaluation in `normalize()`
- fixed bug in `center.group_df()` when `force = TRUE`

### CODE REVIEW

**Comment by strengejacke** [comment]
Need to add tests for `normalize.data.frame(append = TRUE)` and `rescale.data.frame(append = TRUE)`.

File: R/data_rescale.R [hunk 1]
```diff
@@ -202,6 +222,21 @@ rescale.data.frame <- function(x,
     verbose = verbose
   )
 
+  # when we append variables, we call ".pro6cess_append()", which will
```
Comment (etiennebacher) [inline R/data_rescale.R:225]:
```suggestion
  # when we append variables, we call ".process_append()", which will
```

File: tests/testthat/test-normalize.R [hunk 2]
```diff
@@ -167,6 +173,24 @@ test_that("normalize (grouped data)", {
   expect_identical(datawizard, manual)
 })
 
+test_that("normalize (grouped data)z, with append", {
```
Comment (etiennebacher) [inline tests/testthat/test-normalize.R:176]:
```suggestion
test_that("normalize (grouped data), with append", {
```

**Review by etiennebacher — APPROVED** [review]
LGTM, thanks!

**Pull request merged**